testing/cypress/e2e/tests/user-content-access.cy.js

```diff
@@ -69,13 +67,13 @@ describe('Test User Role Access to Content Moderation', () => {
       expect(response.status).to.eq(200)})
 
     // Can update the content moderation settings
-    cy.request('/admin/people/permissions').then((response) => {
+    cy.request('/admin/config/workflow/workflows/manage/publishing_content').then((response) => {
       expect(response.status).to.eq(200)})
 
       cy.logout()
   })
 
-  it('Test Content Manager', () => {
+  it.only('Test Content Manager', () => {
```

Comment:
Why did we update to be `only`? I think this is preventing the site admin test to run.

Reply:
Sorry about that!  i did that to just check the one test to run forgot to remove it.  should be good now!